vulnerabilities/scraper/rust.py

```diff
@@ -0,0 +1,90 @@
+import json
+import pytoml as toml
+from urllib.request import urlopen
+import urllib.request
+from urllib.error import HTTPError
+from io import BytesIO
+from zipfile import ZipFile
+from dephell_specifier import RangeSpecifier
+
+RUSTSEC_DB_URL = 'https://github.com/RustSec/advisory-db/archive/master.zip'
+
+
+def rust_crate_advisories(url, prefix='advisory-db-master/crates/'):
+    with urlopen(url) as response:
+        with ZipFile(BytesIO(response.read())) as zf:
+            for path in zf.namelist():
+                if path.startswith(prefix) and path.endswith('.toml'):
+                    yield toml.load(zf.open(path))
+
+
+def all_versions_of_crate(crate):
+    info_url = "https://crates.io//api/v1/crates/{}".format(crate)
```

Comment:
nit: There's an extra `/` after `.io`.

Reply:
Fixed , btw any idea why it worked while using wrong url?

Reply:
I think URLs with multiple consecutive slashes in the path component are valid (see [this StackOverflow thread](https://stackoverflow.com/questions/20523318/is-a-url-with-in-the-path-section-valid)). That's why I included the "nit" prefix. :)